features/step_definitions/finance/invoicing_steps.rb

```diff
@@ -166,8 +166,8 @@
   assert_equal 1, nodes.count
 end
 
-Then "I should see the list of years with invoices have the following years:" do |table|
-  actual_years = find('#search_year').find_all('option').map(&:value).map(&:to_i)
-  expected_years = table.raw.flatten.map(&:to_i)
+Then "the years selector should have the following options:" do |table|
+  actual_years = find('#search_year').find_all('option').map(&:value).map(&:to_s)
```

Comment:
Doesn't really affect much, because empty string will be transformed to `0`, but I think we shouldn't do this transformation, and just compare the strings.